manifests/install.pp

```diff
@@ -3,13 +3,20 @@
 class redis::install(
   $redis_dl_dir      = '/opt/redis-dl',
   $redis_install_dir = '/opt/redis',
-  $version           = hiera('redis_version', '2.8.19')
+  $version           = hiera('redis_version', '2.8.19'),
+  $sentinel_version   = hiera('sentinel_version', undef)
   ){
 
   include redis::params
   include gcc
   include wget
 
+  if $sentinel_version {
+    $version = $sentinel_version
+  } else {
+    $version = $redis_version
```

Comment:
Above on line 6 the redis_version hiera param is named version. Did you mean to make line 6 redis_version?